jdbc-v2/src/main/java/com/clickhouse/jdbc/ConnectionImpl.java

```diff
@@ -324,19 +323,31 @@ public PreparedStatement prepareStatement(String sql, String[] columnNames) thro
     @Override
     public Clob createClob() throws SQLException {
         checkOpen();
-        throw new SQLFeatureNotSupportedException("Clob not supported");
```

Comment:
I think, we still need to throw exception as we do not support it and old driver either

Reply:
just for reference - https://docs.oracle.com/javadb/10.10.1.2/ref/rrefclob.html
it can be added for example when we allow reading from a file, but today, I think it is not a widely used